generated/python/gapic-google-cloud-datastore-v1/docs/conf.py

```diff
@@ -294,17 +289,15 @@
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
 
-# If true, do not generate a @detailmenu in the "Top" node's menu.
+# If true, do not generate a @etailmenu in the "Top" node's menu.
```

Comment:
This is a bug in toolkit; the `@` needs to be escaped because it is a special character in the snippet language. I missed this when copying the template over from packman. Will put up a PR to do this in a minute